### PR TITLE
Dp 7086 login and help

### PR DESCRIPTION
**Pull request by aditudorache**

### CODE REVIEW

File: src/app/components/Messages/NoResultsForSearchType.jsx [hunk 1]
```diff
@@ -1,28 +1,16 @@
...
-  )
   return (
     <div className="c-link__wrapper--inine-block">
       <div className="u-margin__bottom--1">Geen resultaten van deze soort</div>
       {message}
       {authMessage && (
         <span>
           &nbsp;Medewerkers/ketenpartners van Gemeente Amsterdam kunnen inloggen om meer gegevens te
-          zien, zie {link}
+          zien. <HelpLink />.
```
Comment (royderks) [inline src/app/components/Messages/NoResultsForSearchType.jsx]: Shouldn't there be a LoginLink here as well?

File: src/app/components/PanelMessages/LoginLink/LoginLink.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from '@datapunt/asc-core'
+import { connect } from 'react-redux'
+import { authenticateRequest } from '../../../../shared/ducks/user/user'
+
+const StyledLink = styled.a`
```
Comment (royderks) [inline src/app/components/PanelMessages/LoginLink/LoginLink.jsx]: Why not use the Link component from asc-ui?

File: src/app/components/PanelMessages/MoreResultsWhenLoggedIn.test.jsx [hunk 3]
```diff
@@ -12,6 +12,11 @@ describe('MoreResultsWhenLoggedIn', () => {
     const excludedResults = 'Lorem ipsum'
     const component = shallow(<MoreResultsWhenLoggedIn excludedResults={excludedResults} />)
 
-    expect(component.find('p').text()).toContain(': Lorem ipsum.')
+    expect(
+      component
+        .find('p')
+        .at(0)
+        .text(),
+    ).toContain(': Lorem ipsum.')
```
Comment (royderks) [inline src/app/components/PanelMessages/MoreResultsWhenLoggedIn.test.jsx]: Test for the login/help link?

File: src/app/components/Messages/NoResultsForSearchType.jsx (hunk 1 above)
Comment (aditudorache) [reply src/app/components/Messages/NoResultsForSearchType.jsx]: absolutely! thanks!

File: src/app/components/PanelMessages/LoginLink/LoginLink.jsx (hunk 2 above)
Comment (aditudorache) [reply src/app/components/PanelMessages/LoginLink/LoginLink.jsx]: done

**Pull request merged**